app/services/utm_service.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class UtmService
+  CAMPAIGNS = {
+    new_mentor: "new-mentor",
+    new_early_career_teacher: "new-early-career-teacher",
+    sign_in: "sign-in",
+  }.freeze
+
+  def self.email(campaign)
+    {
+      utm_source: "cpdservice",
+      utm_medium: "email",
+      utm_campaign: CAMPAIGNS[campaign] || "none",
+    }
+  end
```

Comment:
I would ask Chris how he wants us to handle this stuff so it is not mistaken with R&P. I would change the source to something like `cpd-el-service`, but I don't know his setup.

Reply:
No problem, the Jira ticket had cpdservice but will double check with him.

Reply:
Chris is happy with the campaign names. He's asked to keep it as cpdservice for now too. Apparently it doesn't matter to much at the minute what the source name is as they are being tracked individually anyway.